normandy/recipes/api/views.py

```diff
@@ -70,6 +78,28 @@ class RecipeViewSet(CachingViewsetMixin, UpdateOrCreateModelViewSet):
         AdminEnabledOrReadOnly,
     ]
 
+    def get_queryset(self):
```

Comment:
I'll add tests. We could potentially use generic filtering but I'm not sure if it would handle the complex many to many filtering we're doing correctly.